### ISSUE TITLE
how to open/close from parent component.

### ISSUE THREAD

**Issue by hasankemaldemirci** [krystalcampioni/vue-hotel-datepicker#25]
I want to open/close this from parent component.

I have tried this but it doesn't work 
```js
<button @click="$refs.datepicker.isOpen = true">
<HotelDatePicker ref="datepicker"></HotelDatePicker>
```

Is there anyway to do it?

**Comment by chientinhte** [comment]
I solved this: 
let datepicker = $('.datepicker');
datepicker.removeClass('datepicker--closed');
datepicker.addClass('datepicker--open');

**Comment by krystalcampioni** [comment]
@hasankemaldemirci the default behaviour of closing the datepicker when clicking outside of the component was preventing the datepicker from being opened externally. Should be fixed now at v2.3.0. 

To use the open/hide/toggle methods make sure to set the new `closeDatepickerOnClickOutside` prop to `false`
